RetailCoder.VBE/Refactorings/Rename/RenameEventHandler.cs

```diff
@@ -0,0 +1,40 @@
+﻿using Rubberduck.Common;
+using Rubberduck.Parsing.Symbols;
+using Rubberduck.UI;
+using System.Diagnostics;
+using System.Linq;
+
+namespace Rubberduck.Refactorings.Rename
+{
+    public class RenameEventHandler : RenameHandlerBase
```

Comment:
@retailcoder I appreciate the detailed explanation.  Implementing IRefactoring with the 'handlers' sounds great.  I'm still ruminating on how to approach this.